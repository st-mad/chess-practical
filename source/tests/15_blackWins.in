c1
h6
a8
a1
f1
a6
h8
h6
h1
b1
h6
a6
b1
b7
c8
b7
quit
