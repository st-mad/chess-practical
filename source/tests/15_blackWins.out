  a b c d e f g h
8 r . b . . b . r 8
7 . . . . . . . . 7
6 . . . . . . . . 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 R . B . . B . R 1
  a b c d e f g h

White plays. Enter move:
  a b c d e f g h
8 r . b . . b . r 8
7 . . . . . . . . 7
6 . . . . . . . B 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 R . . . . B . R 1
  a b c d e f g h

Black plays. Enter move:
  a b c d e f g h
8 . . b . . b . r 8
7 . . . . . . . . 7
6 . . . . . . . B 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 r . . . . B . R 1
  a b c d e f g h

White plays. Enter move:
  a b c d e f g h
8 . . b . . b . r 8
7 . . . . . . . . 7
6 B . . . . . . B 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 r . . . . . . R 1
  a b c d e f g h

Black plays. Enter move:
  a b c d e f g h
8 . . b . . b . . 8
7 . . . . . . . . 7
6 B . . . . . . r 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 r . . . . . . R 1
  a b c d e f g h

White plays. Enter move:
  a b c d e f g h
8 . . b . . b . . 8
7 . . . . . . . . 7
6 B . . . . . . r 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 r R . . . . . . 1
  a b c d e f g h

Black plays. Enter move:
  a b c d e f g h
8 . . b . . b . . 8
7 . . . . . . . . 7
6 r . . . . . . . 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 r R . . . . . . 1
  a b c d e f g h

White plays. Enter move:
  a b c d e f g h
8 . . b . . b . . 8
7 . R . . . . . . 7
6 r . . . . . . . 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 r . . . . . . . 1
  a b c d e f g h

Black plays. Enter move:
  a b c d e f g h
8 . . . . . b . . 8
7 . b . . . . . . 7
6 r . . . . . . . 6
5 . . . . . . . . 5
4 . . . . . . . . 4
3 . . . . . . . . 3
2 . . . . . . . . 2
1 r . . . . . . . 1
  a b c d e f g h

Black wins!
